system_tests/utils.go

```diff
@@ -741,6 +764,13 @@ func publishAndConsumeMQTTMsg(hostname, nodePort, username, password string, ove
 		SetClientID("system tests MQTT plugin").
 		SetProtocolVersion(4) // RabbitMQ MQTT plugin targets MQTT 3.1.1
 
+	if tlsConfig != nil {
+		url = fmt.Sprintf("ssl://%s:%s", hostname, nodePort)
+		opts = opts.
+			AddBroker(url).
+			SetTLSConfig(tlsConfig)
+	}
+
```

Comment:
Can we also add a test for WebMQTT and TLS, i.e. testing `wss`?

Reply:
I don't see too much value in us testing every single plugins protocol with TLS. I think the coverage we have currently is enough, and would prefer to leave the system tests as it is at least for now.

Reply:
The additional value is that we configured MQTT Web Stomp TLS correctly, i.e. the changes in https://github.com/rabbitmq/cluster-operator/pull/451/files#diff-8963e26fa961496c5fbf37e06487c020529c5cee6f6ff953ed8d63c018d1c5efR115-R128 and that we expose the port. We do have unit tests, but an end-to-end test adds value from my point of view that this feature actually works.

However, as discussed in standup, let's not block merging the PR on this test. @ablease said he will add a test once this is merged.